docs/src/integrate/dxe_core.md

```diff
@@ -330,6 +330,90 @@ Note:
 - Inside `efi_main` we set the global logger to our static logger with the `log` crate and set the maximum log level.
 - The `serial_logger` provides a simple, lightweight logging solution that writes directly to the serial port.
 
+### 6.3 Logging Format Options and Binary Size Impacts
+
+Patina logging features configurable log message formats. These formats control which specific metadata is included with each log entry.
+The format selected has a direct impact on binary size due to embedding string literals and debug information such as line numbers
+and file pathnames throughout the firmware image.
+
+The `patina::log::Format` enum (as defined in `patina/sdk/patina/src/log.rs`) has three format options:
+#### Available Log Format Options
+
+| Format | Metadata Included | Example Output | Binary Size Effect |
+|--------|-------------------|----------------|--------------------|
+| **`Standard`** | Level, message (line/file only for TRACE) | `INFO - Initializing DXE Core` | **Minimal** - Line numbers and file pathnames only for TRACE level |
+| **`Json`** | Level, message | `{"level": "INFO" "message": "Initializing DXE Core"}` | **Minimal** - No debug metadata|
+| **`VerboseJason`** | Level, target, message, file path, line number | `{"level": "INFO", "target": "patina_dxe_core", "message": "Initializing DXE Core", "file": "src/main.rs", "line": 42}` |  **Significant** - Line numbers and full file pathnames for all log statements |
+
+#### Log Format Configuration
+
+Log format is specified whtn creating the logger instance:
+
+```rust
+use patina::log::serial_logger::SerialLogger;
+use patina::log::Format
+use patina::serial::uart::Uart16550;
+
+// Recommended for production code - minimal size impact
+static LOGGER: SerialLogger<Uart16550> = SerialLogger::new(
+    Format::Standard,
+    &[], // Override specific modules here
+    log::LevelFilter::Info,
+    Uart16550::io { base: 0x402 },
+)
+
+For debug and development, `VerboseJason` provides full debug info
+
+```rust
+// Full debug information - recommended only for development
+static LOGGER: SerialLogger<Uart16550> = SerialLogger::new(
+    Format::VerboseJson, // Include all line numbers and file paths
+    &[], // No module overrides
+    log::LevelFilter::Trace,
+    Uart16550::Io { base: 0x402 },
+);
+```
+
+#### Binary Size Considerations
+
+When line numbers are file paths are included in log messages, the Rust compiler embeds these as string literals in the final binary. The total size impact will thus be the sum of all log statements in the codebase.
```

Comment:
As noted above, the string literals are present regardless of the `patina::log::Format` you use. It depends on the usage of `-Zlocation-detail` or not.